core/spec/models/spree/order_updater_spec.rb

```diff
@@ -57,9 +56,9 @@ module Spree
         # A line item will not have both additional and included tax,
         # so please just humour me for now.
         order.line_items.first.update_columns({
-          :adjustment_total => 10.05,
-          :additional_tax_total => 0.05,
-          :included_tax_total => 0.05,
+          adjustment_total: 10.05,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.